templates/common/.vscode/setup-web.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+function codespacesPortUrl() {
+    local portNumber=$1
+    gh codespace ports \
```

Comment:
Does this even work if the service isn't yet running? Doesn't that mean nothing is running at that port?

Reply:
Yes. This communicates with GitHub and sets routing configuration. These settings can be altered even if the port is not currently in use.